static/app/components/alert.tsx

```diff
@@ -52,6 +59,8 @@ function Alert({
         return <IconCheckmark />;
       case 'error':
         return <IconNot />;
+      case 'performanceError':
```

Comment:
maybe performance is too specific since this is supposed to be a generic component

Reply:
Maybe just  `flamingError` or `fireError` :)